editor/src/core/model/element-metadata-utils.ts

```diff
@@ -184,7 +184,7 @@ export const MetadataUtils = {
     const elementMetadata = MetadataUtils.findElementByElementPath(jsxMetadata, path)
     return MetadataUtils.isProbablySceneFromMetadata(elementMetadata)
   },
-  getViewZIndexFromMetadata(metadata: ElementInstanceMetadataMap, target: ElementPath): number {
+  getIndexInParent(metadata: ElementInstanceMetadataMap, target: ElementPath): number {
     const siblings = MetadataUtils.getSiblingsUnordered(metadata, target)
```

Comment:
did you forget to push something? :)

Reply:
🙈 (https://github.com/concrete-utopia/utopia/pull/3534/commits/39fec13a33adf95dd62fad35831043d31acfb7cc)